packages/desktop-client/src/components/budget/tracking/budgetsummary/BudgetTotal.tsx

```diff
@@ -53,8 +52,9 @@ export function BudgetTotal<
         <Text>
           <CellValue binding={current} type="financial" />
```

Comment:
Wondering, would it work to move the `Trans` outside of the `Text`? Otherwise I'm thinking it could be hard to represent languages where the direction is normally inverted, e.g. [Korean](https://translate.google.com/?sl=en&tl=ko&text=5%20of%2020&op=translate)

Reply:
Sorry, should have commented a line lower so this was clearer. I'm worried about languages where the following would need to be inverted:

```tsx
{{ usedAmount: '' }} of {{ allocatedAmount: '' }}
```
With `allocatedAmount` on the left instead

Reply:
I was hoping that by keeping the variables in the translation that Weblate would let them put it the other way round

<img width="1025" alt="image" src="https://github.com/user-attachments/assets/e9fe914e-9150-4630-b7c4-f2dbab259733" />

it seems to let you put the variables wherever you'd like.

Reply:
Right, that's true, but since these are placeholders and not the actual numbers, won't that not actually affect how Actual displays them?

Reply:
🤦‍♂️ doh. yes, it will.

I'll see if I can do it properly

Reply:
what do you think of c613d293f508bee894dc34a163d59e307bec23aa?